pubspec.yaml

```diff
@@ -61,6 +61,7 @@ flutter:
   # To add assets to your application, add an assets section, like this:
   assets:
     - assets/images/logo.svg
+    - assets/images/Man1.png
```

Comment:
pls, add the meaningful name to man's picture. Like login_man_picture. or something else